docs/blitz/design/actions/actions_list.rst

```diff
@@ -26,7 +28,12 @@ exists in the output. You also, have the option to check if a specific
             - 'Should not be in the output'
 
 
-Both include and exclude keywords are optional to use.
+Both include, exclude and connection_alias keywords are optional to use.
+But to use connection_alias two things needs to be followed:
+    1. New mapping datafile schema.
+    2. ``alias`` in mapping datafile should be the ``connection_alias``.
+How to use the new schema for different connection can be found at this `link
+<https://pubhub.devnetcloud.com/media/genie-docs/docs/cookbooks/harness.html#what-can-you-do-with-the-mapping-datafile>`__.
```

Comment:
just comment. very good to have pointer like this!

Reply:
Thank you takashi